### PR TITLE
readme: add more information about baur 2.0 and other improvements

### PR DESCRIPTION
**Pull request by fho**
- more baur 2.0 informations
- restructuring
- a GIF that demonstrates baur will be added in a following PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,45 +2,72 @@
...
+### Key Features
+
+* **Running Tasks only for Changed Applications** \
+  Tasks define which inputs change the result of the task execution. \
+  baur can only runs tasks that have not been run before for the set of inputs.
+  \
+  Inputs can be defined as 
+  [glob file patterns](https://en.wikipedia.org/wiki/Glob_(programming)),
+  as strings on the commandline and the source code files for Go-Applications
+  can be discovered automatically.
```
Comment (FranciscoKurpiel) [inline README.md:38]: Sentence looks a bit strange to me.

**Review by FranciscoKurpiel — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (fho) [reply README.md]: thanks, I changed it

**Comment by nocive** [comment]
late to the party :(

**Comment by fho** [comment]
@nocive we can still make a follow-up PR if you have suggestions for improvements :-)

**Pull request merged**